barretenberg/cpp/src/barretenberg/vm2/tracegen/lib/instruction_spec.cpp

```diff
@@ -69,6 +69,8 @@ const std::unordered_map<ExecutionOpCode, SubtraceInfo> SUBTRACE_INFO_MAP = {
     // Poseidon2Perm
     { ExecutionOpCode::POSEIDON2PERM, { .subtrace_selector = SubtraceSel::POSEIDON2PERM, .subtrace_operation_id = 0 } },
     // Execution
+    // TODO(dbanks12): is it okay that this has subop id 0? This section started with SET at op id 1 before.
```

Comment:
They should probably start with 0, I think Jean is doing some changes in his PR.